packages/mds-policy-service/service/provider.ts

```diff
@@ -99,5 +100,15 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService, PolicyService
       }
 
       return publishedPolicy
+    }),
+  writePolicyIntentToPolicy: (context, intent_draft) =>
+    serviceErrorWrapper('writePolicyIntentToPolicy', async () => {
+      const { policy_id } = await PolicyRepository.writePolicy(translateIntentToPolicy(intent_draft)
```

Comment:
Ah I like this pattern, consolidates all of the translation logic in one place and means we don't have to create an RPC endpoint for every policy intent